src/Outsystems.SetupTools/Functions/Get-OSServerPreReqs.ps1

```diff
@@ -0,0 +1,309 @@
+function Get-OSServerPreReqs
```

Comment:
@fmalheiro, @luislopes-outsystems Think we need to discuss this function. 
For me this should return an object with the pre-reqs installation status. IIS installed (true/false), .NET (true/false and so on)
Then the consumer ( Get-OutSystemsInstalledPrerequisites.ps1 ) should get that object and output whatever it needs to output to the installer window. 
This module should be agnostic and not tailored to a specific use case (ex: OutSystems installer).